docker-new/pdo_services.dockerfile

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------------------------------------------------
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+FROM pdo_services_base
+
+# -----------------------------------------------------------------
+# set up the PDO sources
+# -----------------------------------------------------------------
+ARG REBUILD 0
+
+ARG PDO_DEBUG_BUILD=0
+ENV PDO_DEBUG_BUILD=${PDO_DEBUG_BUILD}
+
+ARG PDO_LEDGER_TYPE=ccf
+ENV PDO_LEDGER_TYPE=${PDO_LEDGER_TYPE}
+
+ARG PDO_INTERPRETER=wawaka
+ENV PDO_INTERPRETER=${PDO_INTERPRETER}
+
+ARG WASM_MEM_CONFIG=MEDIUM
+ENV WASM_MEM_CONFIG=${WASM_MEM_CONFIG}
+
+# copy the source files into the image
+WORKDIR /project/pdo
+COPY --chown=${UNAME}:${UNAME} repository /project/pdo/src
+
+# copy the tools because we want to be able to
+# use them even without a mount point after the
+# container is created
+WORKDIR /project/pdo/tools
+COPY --chown=${UNAME}:${UNAME} tools/*.sh ./
+
+# built it!
+RUN /project/pdo/tools/build_services.sh
+
+# Network ports for running services
```

Comment:
would be good to have a docker image that just builds the services image, and does not start anything. Ideally, we will host this image. Everything from line 50 and below could be part of a conf file  provided externally by a user of the image. For CI, we can have a pdo_services_deploy.dockerfile that starts with this dockerfile (till 50) and does the rest.

Reply:
also will make it easy to star multiple containers for services, starting with the same build image.